File: python/11a_617A_Elephant.py
def main():
    x = int(input())
    pos = 0
    steps = 0
    while pos < x:
        if pos + 5 <= x:
            steps += 1
            pos += 5
        elif pos + 4 <= x:
            steps += 1
            pos += 4
        elif pos + 3 <= x:
            steps += 1
            pos += 3
        elif pos + 2 <= x:
            steps += 1
            pos += 2
        else:
            steps += 1
            pos += 1
    print(steps)

if __name__ == "__main__":
    main()

File: cpp/11a_617A_Elephant.cpp
#include<bits/stdc++.h>

int main(){
    int x, pos = 0, steps = 0;
    std::cin>>x;
    while (pos < x){
        if (pos + 5 <= x){
            pos += 5;
            steps ++;
        }
        else if(pos + 4 <= x){
            pos += 4;
            steps++;
        }
        else if(pos + 3 <= x){
            pos += 3;
            steps ++;
        }
        else if (pos + 2 <= x){
            pos += 2;
            steps ++;
        }
        else{
            pos += 1;
            steps ++;
        }
    }
    std::cout<<steps;
    return 0;
}